src/main/java/com/google/sps/servlets/ImageServlet.java

```diff
@@ -125,11 +148,10 @@ private BlobKey getBlobKey(HttpServletRequest request, String formInputElementNa
     AnnotateImageResponse imageResponse = imageResponses.get(0);
 
     if (imageResponse.hasError()) {
-      System.err.println("Error getting image labels: " + imageResponse.getError().getMessage());
+      System.err.println(String.format("Error getting image %s: %s ", (useTextDetection) ? "text" : "labels", imageResponse.getError().getMessage()));
```

Comment:
[Nit] Please please this into shorter lines.  

I would use a local variable for the ternary statement result to make this a bit more readable as well.